VirtualLibrary/Forms/FaceRecognitionLogin.designer.cs

```diff
@@ -80,7 +79,7 @@ private void InitializeComponent()
             this.loginButton.Name = "loginButton";
             this.loginButton.Size = new System.Drawing.Size(295, 42);
             this.loginButton.TabIndex = 5;
-            this.loginButton.Text = Translations.GetTranslatedString("logInButton");
+            this.loginButton.Text = "Prisijungti kaip:";
```

Comment:
Hey, could you please only commit files or lines that you have changed? 
I think this file is generated automatically because of the size of your computer screen and it also changes the text. As the result - changed text is not being  translated anymore and my work disappears.